addons/web_editor/static/src/components/media_dialog/file_selector.js

```diff
@@ -126,7 +126,10 @@ export class FileSelectorControlPanel extends Component {
             return;
         }
         await this.props.uploadFiles(inputFiles);
-        this.fileInput.el.value = '';
+        const fileInputEl = this.fileInput.el;
+        if (fileInputEl) {
+            fileInputEl.value = "";
+        }
```

Comment:
I think that this should also be done in the `html_editor` module (same file name: `file_selector.js`).

Reply:
Indeed it should have been web_editor only somehow it skipped my mind. Thanks! Let me make those changes ASAP!

Reply:
> I think that this should also be done in the `html_editor` module (same file name: `file_selector.js`).

@bvr-odoo Regarding this point since the fix has been made in v16, we dont have such file for `html_editor` module, but yeah for master I will definitely adapt those changes, Also should it be done in this PR while forward porting or for master should I make new PR ? 
And i have updated the commit message as well.
Thanks!

Reply:
Ok @shsa-odoo my bad I tested it on Master 
We'll adapt it on the forward port (same PR)
Thanks for your change 👍🏼